tests/app/test_retriever.py

```diff
@@ -143,7 +143,7 @@ def create_threads(self, no_of_threads=1, add_internal_draft=False, add_responde
             msg_id = str(uuid.uuid4())
             thread_id = msg_id
             threads.append(thread_id)
-            self.add_secure_message(msg_id=msg_id, thread_id=thread_id, survey=constants.BRES_USER, from_internal=False)
+            self.add_secure_message(msg_id=msg_id, thread_id=thread_id, survey=constants.NON_SPECIFIC_INTERNAL_USER, from_internal=False)
```

Comment:
The survey should be a survey id rather than a user, it looks like this has always been wrong as the user was `BRES` which looks like a survey identifier.